src/test/java/seedu/address/testutil/EditRoomDescriptorBuilder.java

```diff
@@ -53,4 +53,11 @@ public EditRoomDescriptorBuilder withPatient(Name name) {
     public EditRoomDescriptor build() {
         return descriptor;
     }
+
+    @Override
```

Comment:
Just curious why we need this toString() method. Is it for testing haha? But can keep it anyway.

Reply:
Yup, my test cases were failing, so I added a toString() to check. I thought about removing it, but I figured you might need it. Feel free to delete later. Don't think it matters

Reply:
Yep no worries!